SHiELD/coupler_main.F90

```diff
@@ -388,12 +385,16 @@ subroutine coupler_init
 
     call print_memuse_stats('after atmos model init')
 
-    call mpp_get_global_domain(Atm%Domain, xsize=gnlon, ysize=gnlat)
-    allocate ( glon_bnd(gnlon+1,gnlat+1), glat_bnd(gnlon+1,gnlat+1) )
-    call mpp_global_field(Atm%Domain, Atm%lon_bnd, glon_bnd, position=CORNER)
-    call mpp_global_field(Atm%Domain, Atm%lat_bnd, glat_bnd, position=CORNER)
-
-    if (.NOT.Atm%bounded_domain) call data_override_init (Atm_domain_in  = Atm%domain)
+    if (file_exists('data_table') .and. .NOT.Atm%bounded_domain) then
```

Comment:
Does this need to work with the data_table.yaml?  Does this coupler version need to be updated @uramirez8707 ?